src/detect-uricontent.c

```diff
@@ -1743,18 +934,7 @@ static int DetectUricontentIsdataatParseTest(void)
 
 static void DetectUricontentRegisterTests(void)
 {
-    UtRegisterTest("HTTPUriTest01", HTTPUriTest01);
-    UtRegisterTest("HTTPUriTest02", HTTPUriTest02);
-    UtRegisterTest("HTTPUriTest03", HTTPUriTest03);
-    UtRegisterTest("HTTPUriTest04", HTTPUriTest04);
-
-    UtRegisterTest("DetectUriSigTest01", DetectUriSigTest01);
-    UtRegisterTest("DetectUriSigTest02", DetectUriSigTest02);
-    UtRegisterTest("DetectUriSigTest03", DetectUriSigTest03);
     UtRegisterTest("DetectUriSigTest04 - Modifiers", DetectUriSigTest04);
-    UtRegisterTest("DetectUriSigTest05 - Inspection", DetectUriSigTest05);
-    UtRegisterTest("DetectUriSigTest06 - Inspection", DetectUriSigTest06);
-    UtRegisterTest("DetectUriSigTest07 - Inspection", DetectUriSigTest07);
     UtRegisterTest("DetectUriSigTest08", DetectUriSigTest08);
     UtRegisterTest("DetectUriSigTest09", DetectUriSigTest09);
     UtRegisterTest("DetectUriSigTest10", DetectUriSigTest10);
```

Comment:
Can you please renumber the tests here, so we don't have them starting at `04` and skipping sequence numbers? :)

Reply:
Ok. I'll do that. I thought it would be better to do the conversion to FAIL/PASS API in another PR to avoid too many changes in one PR. I'll add the conversion to the next Iteration since it's fine to do that.

Reply:
I see! I think we prefer them to have their own commit, but it would be ok in this case to share them in the same PR (especially now that the other changes are already reviewed)